tests/integration/docusaurus/deployment_patterns/aws_cloud_storage_pandas.py

```diff
@@ -0,0 +1,304 @@
+import os
+import pathlib
+import tempfile
+from great_expectations.core.yaml_handler import YAMLHandler
+import boto3
+
+client = boto3.client("s3")
+temp_dir = tempfile.TemporaryDirectory()
+full_path_to_project_directory = pathlib.Path(temp_dir.name).resolve()
+yaml: YAMLHandler = YAMLHandler()
+# <snippet name="tests/integration/docusaurus/deployment_patterns/aws_cloud_storage_pandas.py imports">
+import great_expectations as gx
+
+context = gx.data_context.FileDataContext.create(full_path_to_project_directory)
+# </snippet>
+
+# parse great_expectations.yml for comparison
+great_expectations_yaml_file_path = os.path.join(
+    full_path_to_project_directory, "great_expectations/great_expectations.yml"
+)
+with open(great_expectations_yaml_file_path) as f:
+    great_expectations_yaml = yaml.load(f)
```

Comment:
```suggestion
great_expectations_yaml = yaml.load(great_expectations_yaml_file_path.read_text())
```
Assuming the above is a `pathlib.Path`